multisigcore/providers/insight.py

```diff
@@ -23,11 +26,13 @@ def spendables_for_addresses(self, bitcoin_addresses):
             addresses = bitcoin_addresses[i:i+CHUNK_SIZE]
             url = "%s/api/addrs/%s/utxo" % (self.base_url, ",".join(addresses))
             r.extend(json.loads(urlopen(url).read().decode("utf8")))
-        spendables = []
+        spendables = {}
```

Comment:
I see. Could you make that change in multisig-core? As written, it was failing on send tx.

You're right it's better to stay compatible, Richard might end up merging the function back into pycoin.